.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+    - "9.5"
```

Comment:
The README says we require Node 8+. Should we be building on Node 8 here, or update the README to match this value?

Reply:
I'll change it to test on both "node" (9.x) and "lts" (8.x) to run on the major versions. I don't think we've written enough code to know if we're going to use a library or something that only works in one version or the other.

I'm personally using 9.3 so I'll have the README reflect 9.x until we run into something that tells us the exact version we need to stay on.